web/js/src/trimInput.js

```diff
@@ -0,0 +1,8 @@
+//trim leading / trailing spaces form input fields
+export default function trimInput() {
```

Comment:
Maybe `handleAutoTrimInputs` would be more consistent?

Reply:
Fine for me ..

Reply:
Renamed function to handleAutoTrimInputs and filename to autoTrimInputs
...